internal/exec/engine.go

```diff
@@ -192,6 +212,10 @@ func (e *Engine) renderConfig(cfg types.Config, f resource.Fetcher) (types.Confi
 		// Replace the HTTP client in the fetcher to be configured with the
 		// timeouts of the new config
 		f.UpdateHttpTimeouts(newCfg.Ignition.Timeouts)
+		err = f.UpdateHttpsCAs(newCfg.Ignition.Security.TLS.CertificateAuthorities)
```

Comment:
maybe combine UpdateHttpTimeouts and UpdateHttpsCAs since they are always called together